allennlp/tests/common/params_test.py

```diff
@@ -209,6 +209,9 @@ def test_known_configs(self):
             # dependency parser
             'PTB_DEPENDENCIES_TRAIN', 'PTB_DEPENDENCIES_VAL',
 
+            # multilingual dependency parser
+            'TRAIN_PATHNAME', 'DEV_PATHNAME', 'TEST_PATHNAME'
```

Comment:
You're missing a comma at the end of this, that's probably why the test is failing.